assets/templates/dataset-filter/hierarchy.tmpl

```diff
@@ -67,7 +67,6 @@
 									</ul>
 								</div>
 							{{else}}
-								<input name="save-and-return" class="visuallyhidden" type="submit" value="Save and return" />
```

Comment:
Pressing enter after tabbing to content triggers add all not save and return.
class="visuallyhidden" seems to be the problem and changing to class="hidden" fixes the tabbing.
https://github.com/ONSdigital/sixteens/blob/develop/scss/utilities/_utilities.scss hidden and visuallyhidden css.